[2017-06-27 21:17:40] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:17:40] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:17:40] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@EspritFamycity/Default/home.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:17:41] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:17:41] php.CRITICAL: Uncaught Exception: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:17:41] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:17:41] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:19:00] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:19:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:19:00] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@EspritFamycity/Default/home.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:19:00] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:19:00] php.CRITICAL: Uncaught Exception: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:19:00] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:19:00] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:20:38] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:20:39] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:20:39] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@EspritFamycity/Default/home.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:20:39] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:20:39] php.CRITICAL: Uncaught Exception: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:20:39] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:20:39] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:26:38] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:26:38] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:26:38] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@EspritFamycity/Default/home.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:26:38] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:26:38] php.CRITICAL: Uncaught Exception: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:26:39] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig"." at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:26:39] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Extension\Core.php on line 139 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\vendor\twig\twig\lib\Twig\Environment.php line 549) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Environment.php:549, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Extension\\Core.php on line 139 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:30:40] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:30:40] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:30:40] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@EspritFamycity/Default/home.html.twig"." at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:30:40] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:30:40] php.CRITICAL: Uncaught Exception: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig". {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:30:40] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig"." at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:30:40] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:38:54] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:39:07] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:39:13] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@EspritFamycity/Default/home.html.twig"." at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:39:14] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:39:14] php.CRITICAL: Uncaught Exception: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig". {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:39:14] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig"." at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513 {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35, Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@EspritFamycity/Default/home.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:39:14] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Syntax: An exception has been thrown during the compilation of a template ("Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\wamp64\www\FamicityMah\var\cache\dev\classes.php on line 5811 and defined") in "@Twig/Exception/exception_full.html.twig". at C:\wamp64\www\FamicityMah\var\cache\dev\classes.php line 5513) {"exception":"[object] (Twig_Error_Syntax(code: 0): An exception has been thrown during the compilation of a template (\"Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined\") in \"@Twig/Exception/exception_full.html.twig\". at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php:5513, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Twig_Filter::__construct() must be an instance of string, string given, called in C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\classes.php on line 5811 and defined at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Filter.php:35)"} []
[2017-06-27 21:43:48] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:43:48] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:43:53] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:43:53] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/dist/img/user4-128x128.jpg" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/dist/img/user4-128x128.jpg\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:43:54] php.DEBUG: Warning: mkdir(): File exists {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"C:\\\\wamp64\\\\www\\\\FamicityMah\\\\vendor\\\\twig\\\\twig\\\\lib\\\\Twig\\\\Cache\\\\Filesystem.php\",\"line\":52})"} []
[2017-06-27 21:43:54] php.DEBUG: Warning: rename(C:\wamp64\www\FamicityMah\var\cache\dev\twig\89\8949A66.tmp,C:\wamp64\www\FamicityMah\var\cache\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php): Acc�s refus�. (code: 5) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"C:\\\\wamp64\\\\www\\\\FamicityMah\\\\vendor\\\\twig\\\\twig\\\\lib\\\\Twig\\\\Cache\\\\Filesystem.php\",\"line\":63})"} []
[2017-06-27 21:43:54] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Failed to write cache file "C:\wamp64\www\FamicityMah\var\cache\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php"."). at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Bundle\TwigBundle\Resources\views\layout.html.twig line 26) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php\".\"). at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Resources\\views\\layout.html.twig:26, RuntimeException(code: 0): Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:78)"} []
[2017-06-27 21:43:54] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("Failed to write cache file "C:\wamp64\www\FamicityMah\var\cache\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php"."). {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php\".\"). at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Resources\\views\\layout.html.twig:26, RuntimeException(code: 0): Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:78, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:43:54] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Failed to write cache file "C:\wamp64\www\FamicityMah\var\cache\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php".")." at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Bundle\TwigBundle\Resources\views\layout.html.twig line 26 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php\".\"). at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Resources\\views\\layout.html.twig:26, RuntimeException(code: 0): Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/89/8946063f867366fe039e717f1ea33334e7f4d655c3ccdba48624f4f2ad3903a4.php\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:78, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:43:55] php.DEBUG: Warning: mkdir(): File exists {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"C:\\\\wamp64\\\\www\\\\FamicityMah\\\\vendor\\\\twig\\\\twig\\\\lib\\\\Twig\\\\Cache\\\\Filesystem.php\",\"line\":52})"} []
[2017-06-27 21:43:55] php.DEBUG: Warning: mkdir(): File exists {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"C:\\\\wamp64\\\\www\\\\FamicityMah\\\\vendor\\\\twig\\\\twig\\\\lib\\\\Twig\\\\Cache\\\\Filesystem.php\",\"line\":52})"} []
[2017-06-27 21:43:55] php.DEBUG: Warning: rename(C:\wamp64\www\FamicityMah\var\cache\dev\twig\28\2809D03.tmp,C:\wamp64\www\FamicityMah\var\cache\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php): Acc�s refus�. (code: 5) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"C:\\\\wamp64\\\\www\\\\FamicityMah\\\\vendor\\\\twig\\\\twig\\\\lib\\\\Twig\\\\Cache\\\\Filesystem.php\",\"line\":63})"} []
[2017-06-27 21:43:55] request.CRITICAL: Exception thrown when handling an exception (Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Failed to write cache file "C:\wamp64\www\FamicityMah\var\cache\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php"."). at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Bundle\TwigBundle\Resources\views\Exception\traces.html.twig line 14) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php\".\"). at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Resources\\views\\Exception\\traces.html.twig:14, RuntimeException(code: 0): Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:78)"} []
[2017-06-27 21:43:55] php.CRITICAL: Uncaught Exception: An exception has been thrown during the rendering of a template ("Failed to write cache file "C:\wamp64\www\FamicityMah\var\cache\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php"."). {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php\".\"). at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Resources\\views\\Exception\\traces.html.twig:14, RuntimeException(code: 0): Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:78, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/dist/img/user4-128x128.jpg\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:43:55] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Failed to write cache file "C:\wamp64\www\FamicityMah\var\cache\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php".")." at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Bundle\TwigBundle\Resources\views\Exception\traces.html.twig line 14 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php\".\"). at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Resources\\views\\Exception\\traces.html.twig:14, RuntimeException(code: 0): Failed to write cache file \"C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev/twig/28/280273bc3e911e96600eb30d97ddc8220528803ad01d251f55f1436b6a157938.php\". at C:\\wamp64\\www\\FamicityMah\\vendor\\twig\\twig\\lib\\Twig\\Cache\\Filesystem.php:78, Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/dist/img/user4-128x128.jpg\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:43:57] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"473722","_route":"_wdt"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/_wdt/473722","method":"GET"} []
[2017-06-27 21:43:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:46:48] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:46:48] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:46:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/dist/img/user4-128x128.jpg" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/dist/img/user4-128x128.jpg\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:46:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:46:52] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"c1543d","_route":"_wdt"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/_wdt/c1543d","method":"GET"} []
[2017-06-27 21:46:53] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:49:43] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:49:43] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:49:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/dist/img/user4-128x128.jpg" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/dist/img/user4-128x128.jpg\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:49:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:49:46] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"7d8e62","_route":"_wdt"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/_wdt/7d8e62","method":"GET"} []
[2017-06-27 21:49:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:52:10] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-27 21:52:10] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-27 21:52:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:52:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/dist/img/user4-128x128.jpg" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/dist/img/user4-128x128.jpg\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-27 21:52:13] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"616361","_route":"_wdt"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/_wdt/616361","method":"GET"} []
[2017-06-27 21:52:13] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-28 12:16:42] php.INFO: User Deprecated: The "Symfony\Component\Console\Event\ConsoleExceptionEvent" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): User Deprecated: The \"Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent\" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent.php:14)"} []
[2017-06-28 12:16:42] request.INFO: Matched route "esprit_famycity_homepage". {"route":"esprit_famycity_homepage","route_parameters":{"_controller":"Esprit\\FamycityBundle\\Controller\\DefaultController::homeAction","_route":"esprit_famycity_homepage"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/","method":"GET"} []
[2017-06-28 12:16:42] console.ERROR: Error thrown while running command "doctrine:schema:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)","command":"doctrine:schema:create","message":"An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue"} []
[2017-06-28 12:16:42] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-28 12:16:42] console.ERROR: Command "doctrine:schema:create" exited with code "1" {"command":"doctrine:schema:create","code":1} []
[2017-06-28 12:16:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/dist/img/user4-128x128.jpg" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/dist/img/user4-128x128.jpg\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-28 12:16:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js" (from "http://localhost/FamicityMah/web/app_dev.php/")" at C:\wamp64\www\FamicityMah\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /Famicity/https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.min.js\" (from \"http://localhost/FamicityMah/web/app_dev.php/\") at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp64\\www\\FamicityMah\\var\\cache\\dev\\appDevDebugProjectContainerUrlMatcher.php:349)"} []
[2017-06-28 12:16:50] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"099b15","_route":"_wdt"},"request_uri":"http://localhost/FamicityMah/web/app_dev.php/_wdt/099b15","method":"GET"} []
[2017-06-28 12:16:50] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-06-28 12:18:05] php.INFO: User Deprecated: The "Symfony\Component\Console\Event\ConsoleExceptionEvent" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): User Deprecated: The \"Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent\" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent.php:14)"} []
[2017-06-28 12:18:05] console.ERROR: Error thrown while running command "doctrine:schema:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)","command":"doctrine:schema:create","message":"An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue"} []
[2017-06-28 12:18:05] console.ERROR: Command "doctrine:schema:create" exited with code "1" {"command":"doctrine:schema:create","code":1} []
[2017-06-28 12:18:28] php.INFO: User Deprecated: The "Symfony\Component\Console\Event\ConsoleExceptionEvent" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): User Deprecated: The \"Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent\" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent.php:14)"} []
[2017-06-28 12:18:28] console.ERROR: Error thrown while running command "doctrine:schema:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)","command":"doctrine:schema:create","message":"An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue"} []
[2017-06-28 12:18:28] console.ERROR: Command "doctrine:schema:create" exited with code "1" {"command":"doctrine:schema:create","code":1} []
[2017-06-28 12:18:36] php.INFO: User Deprecated: The "Symfony\Component\Console\Event\ConsoleExceptionEvent" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): User Deprecated: The \"Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent\" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent.php:14)"} []
[2017-06-28 12:18:36] console.ERROR: Error thrown while running command "doctrine:schema:create". Message: "An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)","command":"doctrine:schema:create","message":"An exception occured in driver: SQLSTATE[HY000] [1049] Base 'fcitydb' inconnue"} []
[2017-06-28 12:18:36] console.ERROR: Command "doctrine:schema:create" exited with code "1" {"command":"doctrine:schema:create","code":1} []
[2017-06-28 12:19:03] php.INFO: User Deprecated: The "Symfony\Component\Console\Event\ConsoleExceptionEvent" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): User Deprecated: The \"Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent\" class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead. at C:\\wamp64\\www\\FamicityMah\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent.php:14)"} []
[2017-06-28 12:19:03] console.ERROR: Error thrown while running command "doctrine:schema:create". Message: "Column name `id` referenced for relation from Esprit\FamycityBundle\Entity\Publication towards Esprit\FamycityBundle\Entity\Commentaires does not exist." {"error":"[object] (Doctrine\\ORM\\ORMException(code: 0): Column name `id` referenced for relation from Esprit\\FamycityBundle\\Entity\\Publication towards Esprit\\FamycityBundle\\Entity\\Commentaires does not exist. at C:\\wamp64\\www\\FamicityMah\\vendor\\doctrine\\orm\\lib\\Doctrine\\ORM\\Tools\\SchemaTool.php:627)","command":"doctrine:schema:create","message":"Column name `id` referenced for relation from Esprit\\FamycityBundle\\Entity\\Publication towards Esprit\\FamycityBundle\\Entity\\Commentaires does not exist."} []
[2017-06-28 12:19:03] console.ERROR: Command "doctrine:schema:create" exited with code "1" {"command":"doctrine:schema:create","code":1} []
[2017-06-28 13:33:30] doctrine.DEBUG: CREATE TABLE Commentaires (id INT AUTO_INCREMENT NOT NULL, owner_id INT DEFAULT NULL, value VARCHAR(255) NOT NULL, UNIQUE INDEX UNIQ_C18F1B3C7E3C61F9 (owner_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-06-28 13:33:30] doctrine.DEBUG: CREATE TABLE event (id_event INT AUTO_INCREMENT NOT NULL, participants_id INT DEFAULT NULL, add_user_id INT DEFAULT NULL, categorie VARCHAR(255) NOT NULL, date_debut DATE NOT NULL, date_fin DATE NOT NULL, libelle VARCHAR(255) NOT NULL, cost DOUBLE PRECISION NOT NULL, INDEX IDX_3BAE0AA7838709D5 (participants_id), UNIQUE INDEX UNIQ_3BAE0AA7955A6137 (add_user_id), PRIMARY KEY(id_event)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-06-28 13:33:31] doctrine.DEBUG: CREATE TABLE Publication (id_publication INT AUTO_INCREMENT NOT NULL, participants_id INT DEFAULT NULL, comments_id INT DEFAULT NULL, createur_id INT DEFAULT NULL, categorie VARCHAR(255) NOT NULL, pub_date DATE NOT NULL, libelle VARCHAR(255) NOT NULL, image_src VARCHAR(255) NOT NULL, video_src VARCHAR(255) NOT NULL, INDEX IDX_29A0E8AE838709D5 (participants_id), INDEX IDX_29A0E8AE63379586 (comments_id), UNIQUE INDEX UNIQ_29A0E8AE73A201E5 (createur_id), PRIMARY KEY(id_publication)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-06-28 13:33:31] doctrine.DEBUG: CREATE TABLE reclamation (id INT AUTO_INCREMENT NOT NULL, titre VARCHAR(100) NOT NULL, objet VARCHAR(100) NOT NULL, description VARCHAR(1000) NOT NULL, date DATE NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-06-28 13:33:32] doctrine.DEBUG: CREATE TABLE utilisateur (id INT AUTO_INCREMENT NOT NULL, username VARCHAR(180) NOT NULL, username_canonical VARCHAR(180) NOT NULL, email VARCHAR(180) NOT NULL, email_canonical VARCHAR(180) NOT NULL, enabled TINYINT(1) NOT NULL, salt VARCHAR(255) DEFAULT NULL, password VARCHAR(255) NOT NULL, last_login DATETIME DEFAULT NULL, confirmation_token VARCHAR(180) DEFAULT NULL, password_requested_at DATETIME DEFAULT NULL, roles LONGTEXT NOT NULL COMMENT '(DC2Type:array)', nom VARCHAR(255) NOT NULL, prenom VARCHAR(255) NOT NULL, UNIQUE INDEX UNIQ_1D1C63B392FC23A8 (username_canonical), UNIQUE INDEX UNIQ_1D1C63B3A0D96FBF (email_canonical), UNIQUE INDEX UNIQ_1D1C63B3C05FB297 (confirmation_token), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-06-28 13:33:32] doctrine.DEBUG: ALTER TABLE Commentaires ADD CONSTRAINT FK_C18F1B3C7E3C61F9 FOREIGN KEY (owner_id) REFERENCES utilisateur (id) [] []
[2017-06-28 13:33:33] doctrine.DEBUG: ALTER TABLE event ADD CONSTRAINT FK_3BAE0AA7838709D5 FOREIGN KEY (participants_id) REFERENCES utilisateur (id) [] []
[2017-06-28 13:33:34] doctrine.DEBUG: ALTER TABLE event ADD CONSTRAINT FK_3BAE0AA7955A6137 FOREIGN KEY (add_user_id) REFERENCES utilisateur (id) [] []
[2017-06-28 13:33:34] doctrine.DEBUG: ALTER TABLE Publication ADD CONSTRAINT FK_29A0E8AE838709D5 FOREIGN KEY (participants_id) REFERENCES utilisateur (id) [] []
[2017-06-28 13:33:35] doctrine.DEBUG: ALTER TABLE Publication ADD CONSTRAINT FK_29A0E8AE63379586 FOREIGN KEY (comments_id) REFERENCES Commentaires (id) [] []
[2017-06-28 13:33:36] doctrine.DEBUG: ALTER TABLE Publication ADD CONSTRAINT FK_29A0E8AE73A201E5 FOREIGN KEY (createur_id) REFERENCES utilisateur (id) [] []
